packages/validator/test/mailto.spec.js

```diff
@@ -0,0 +1,195 @@
+import { test, assert } from './test.js'
+import { access, DID } from '../src/lib.js'
+import { capability, URI, Link, Schema } from '../src/lib.js'
+import { Failure } from '../src/error.js'
+import { ed25519, Verifier } from '@ucanto/principal'
+import * as Client from '@ucanto/client'
+import * as Core from '@ucanto/core'
+
+import { alice, bob, mallory, service } from './fixtures.js'
+const w3 = service.withDID('did:web:web3.storage')
+
+const claim = capability({
+  can: 'access/claim',
+  with: DID.match({ method: 'mailto' }),
```

Comment:
I really don't think we should use `did:mailto` if we don't have to. And I think it's a good sign we don't have to if no one wants to know how to go from `did:mailto` to a full did document. iirc in another thread, I pointed this out, and @Gozala seemed open to just using `mailto:` URIs.

Can we just use mailto uris?

If we really need a did, can we prototype using:
`did:web:web3.storage:account:${encodeURIComponent(`mailto:ben@dag.house`)}}` 

(but IMO we don't need a DID at all here)

Reply:
> I think it's a good sign we don't have to if no one wants to know how to go from did:mailto to a full did document.

We do not want that however, what we want to do is find a proof that we (trusted authority) have authorized user agent identified by some did:key to act on behalf of `did:mailto`'s authority.

It does not mean we resolved DID key or DID document (perhaps we did during authentication), we simply set a cookie in the authenticated user agent so we don't have to reauth as long as our authorization is valid.

Reply:
Created an issue to discuss this further https://github.com/web3-storage/specs/issues/25